Traceback (most recent call last):
  File "train_script.py", line 31, in <module>
    from wrappers_gpt2 import RetrievalGenerationModelGPT2
  File "/scratch/zw2374/public/faiss_db/wrappers_gpt2.py", line 108, in <module>
    from models_gpt2 import RetrievalGPT2Model
  File "/scratch/zw2374/public/faiss_db/models_gpt2.py", line 33, in <module>
    logging.basicConfig(level=logging.INFO)
AttributeError: module 'transformers.utils.logging' has no attribute 'basicConfig'
Fatal error condition occurred in /opt/vcpkg/buildtrees/aws-c-io/src/9e6648842a-364b708815.clean/source/event_loop.c:72: aws_thread_launch(&cleanup_thread, s_event_loop_destroy_async_thread_fn, el_group, &thread_options) == AWS_OP_SUCCESS
Exiting Application
################################################################################
Stack trace:
################################################################################
/ext3/miniconda3/envs/rblm/lib/python3.8/site-packages/pyarrow/libarrow.so.900(+0x200af06) [0x150ec04d3f06]
/ext3/miniconda3/envs/rblm/lib/python3.8/site-packages/pyarrow/libarrow.so.900(+0x20028e5) [0x150ec04cb8e5]
/ext3/miniconda3/envs/rblm/lib/python3.8/site-packages/pyarrow/libarrow.so.900(+0x1f27e09) [0x150ec03f0e09]
/ext3/miniconda3/envs/rblm/lib/python3.8/site-packages/pyarrow/libarrow.so.900(+0x200ba3d) [0x150ec04d4a3d]
/ext3/miniconda3/envs/rblm/lib/python3.8/site-packages/pyarrow/libarrow.so.900(+0x1f25948) [0x150ec03ee948]
/ext3/miniconda3/envs/rblm/lib/python3.8/site-packages/pyarrow/libarrow.so.900(+0x200ba3d) [0x150ec04d4a3d]
/ext3/miniconda3/envs/rblm/lib/python3.8/site-packages/pyarrow/libarrow.so.900(+0x1ee0b46) [0x150ec03a9b46]
/ext3/miniconda3/envs/rblm/lib/python3.8/site-packages/pyarrow/libarrow.so.900(+0x194546a) [0x150ebfe0e46a]
/lib/x86_64-linux-gnu/libc.so.6(+0x49a27) [0x150fbc66aa27]
/lib/x86_64-linux-gnu/libc.so.6(on_exit+0) [0x150fbc66abe0]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xfa) [0x150fbc6480ba]
python(+0x1d6e13) [0x558e9a960e13]
/opt/slurm/data/slurmd/job30537033/slurm_script: line 262: 1144407 Aborted                 singularity exec --nv --overlay /scratch/zw2374/overlay-50G-10M.ext3:ro /scratch/work/public/singularity/cuda11.3.0-cudnn8-devel-ubuntu20.04.sif /bin/bash -c "
source /ext3/env.sh
conda activate rblm
python train_script.py --model_path gpt2 --data_config data_config.json --data_folder fast_processed_data_all_roberta_gpt2 --output roberta_gpt2_not_concat_rerun --epochs 200 --save_head  --save_epochs 1 --external_embedding --test_eval --not_concat_self --lr 1e-5
"
